features/file_fixture.feature

```diff
@@ -36,3 +36,23 @@ Feature: Using `file_fixture`
       """
     When I run `rspec spec/lib/file_spec.rb`
     Then the examples should all pass
+
+  Scenario: Creating a ActiveStorage::Blob from a file fixture
+    Given a file named "spec/fixtures/files/sample.txt" with:
+      """
+      Hello
+      """
+    And a file named "spec/lib/fixture_set_blob.rb" with:
+      """ruby
+      require "rails_helper"
+
+      RSpec.describe "blob" do
+        it "creates a blob from a sample file" do
+          if defined?(ActiveStorage::FixtureSet)
```

Comment:
Let's skip this scenario altogether for pre-7 Rails like we [used to do here with pre-6 features](https://github.com/rspec/rspec-rails/commit/229f5f7c#diff-f57037e7ce9e7f58b722ba688654211d528c6c30ed7cd89df2cae8db9896ad14). That keeps it easier to read, and there's a visual note that this is only for Rails 7.

Reply:
I see. I added the change.